src/PersonObjects/Player/PlayerObjectGeneralMethods.tsx

```diff
@@ -590,7 +590,7 @@ export function process(this: IPlayer, router: IRouter, numCycles = 1): void {
   if (this.isWorking) {
     if (this.workType == CONSTANTS.WorkTypeFaction) {
       if (this.workForFaction(numCycles)) {
-        router.toFaction();
```

Comment:
careful with this, ive noticed references to Factions causes issues when augmenting as Factions[this.currentWorkFactionName] can return null, so the same tick you augment itll then throw the recovery screen.

Reply:
but int his case toFaction has a if check so i think this ones safe